### PR TITLE
feat: Add active element support

### PR DESCRIPTION
**Pull request by vane**
Add support to retrieve active element from android application.
Corresponding PR
https://github.com/appium/appium-uiautomator2-driver/pull/457

Sample response :
```
{
  "value": {
    "element-6066-11e4-a52e-4f735466cecf": "00000000-0000-0009-ffff-ffff0000003c",
    "ELEMENT": "00000000-0000-0009-ffff-ffff0000003c"
  },
  "sessionId": "8cfba1e8-117e-4a34-a4c2-a4846b3010cb",
  "status": 0
}
```

### CODE REVIEW

File: app/src/main/java/io/appium/uiautomator2/handler/ActiveElement.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+public class ActiveElement extends SafeRequestHandler {
+
+    public ActiveElement(String mappedUri) {
+        super(mappedUri);
+    }
+
+    @Override
+    protected AppiumResponse safeHandle(IHttpRequest request) throws UiObjectNotFoundException {
+        refreshAccessibilityCache();
+        Object element = CustomUiDevice.getInstance().findObject(By.focused(true));
```
Comment (mykola-mokhnach) [inline app/src/main/java/io/appium/uiautomator2/handler/ActiveElement.java:42]: Please update the type according to the changes made in https://github.com/appium/appium-uiautomator2-server/pull/422

Comment (mykola-mokhnach) [reply app/src/main/java/io/appium/uiautomator2/handler/ActiveElement.java:42]: Please add handler for the case when no element is found

Comment (vane) [reply app/src/main/java/io/appium/uiautomator2/handler/ActiveElement.java]: It's done

Comment (jzaleski) [reply app/src/main/java/io/appium/uiautomator2/handler/ActiveElement.java]: Great suggestion @mykola-mokhnach -- thanks for the quick turn-around on the fix @vane

**Review by jlipps — APPROVED** [review]

**Review by mykola-mokhnach — APPROVED** [review]

File: app/src/main/java/io/appium/uiautomator2/server/AppiumServlet.java [hunk 2]
```diff
@@ -116,6 +117,7 @@ private void registerDeleteHandler() {
     private void registerPostHandler() {
         register(postHandler, new NewSession("/wd/hub/session"));
         register(postHandler, new FindElement("/wd/hub/session/:sessionId/element"));
+        register(postHandler, new ActiveElement("/wd/hub/session/:sessionId/element/active"));
```
Comment (mykola-mokhnach) [inline app/src/main/java/io/appium/uiautomator2/server/AppiumServlet.java:120]: the handler must be changed to GET, please create a PR with a patch

Comment (jzaleski) [reply app/src/main/java/io/appium/uiautomator2/server/AppiumServlet.java:120]: Thanks for keeping us honest @mykola-mokhnach -- per our offline conversation we'll get it done.

Comment (jzaleski) [reply app/src/main/java/io/appium/uiautomator2/server/AppiumServlet.java:120]: This has been resolved with: https://github.com/appium/appium-uiautomator2-server/pull/424

**Pull request merged**